src/components/SiteFooter/styles.css

```
@@ -0,0 +1,16 @@
+ul {
+	display: flex;
+	gap: 0.5rem;
+	justify-content: center;
+	align-items: center;
+	padding: 0.75rem;
+}
+
+.footerItems{
```

```suggestion
#bottom-footer .footer-items{
```

Add an ID to namespace the items and css is `kebab-case`.